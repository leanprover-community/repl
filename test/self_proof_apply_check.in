{"cmd": "theorem ex : False := by apply ex"}

{"cmd": "theorem ex : False := sorry", "sorries": "individual"}

{"proofState": 0, "tactic": "apply ex"}
