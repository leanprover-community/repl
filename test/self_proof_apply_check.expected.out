{"messages":
 [{"severity": "error",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 10},
   "data":
   "fail to show termination for\n  ex\nwith errors\nfailed to infer structural recursion:\nno parameters suitable for structural recursion\n\nwell-founded recursion cannot be used, `ex` does not take any (non-fixed) arguments"}],
 "env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 1, "column": 22},
   "goal": "‚ä¢ False",
   "endPos": {"line": 1, "column": 27}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 10},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

{"proofStatus":
 "Error: kernel type check failed: (kernel) declaration has free variables '[anonymous]'",
 "proofState": 1,
 "goals": []}

